bayeux.go

```diff
@@ -95,13 +104,31 @@ type AuthenticationParameters struct {
 
 // Bayeux struct allow for centralized storage of creds, ids, and cookies
 type Bayeux struct {
-	creds Credentials
-	id    clientIDAndCookies
+	creds  Credentials
+	id     clientIDAndCookies
+	status Status
+}
+
+func (b *Bayeux) GetConnectedCount() int {
+	return status.connectCount
+}
+
+type Client struct {
+	BayOb Bayeux
```

Comment:
What does `Ob` stand for?

Reply:
Object.

Reply:
let me update it to something meaningful.